composer.json

```diff
@@ -4,7 +4,7 @@
 	"type": "library",
 	"license": "BSD-3-Clause",
 	"require": {
-		"php": ">=5.4"
+		"php": ">=7.0"
```

Comment:
I'm all in but not sure if @JanTvrdik - he was specifically pushing this.

Reply:
Fully understand.

Reply:
I have mostly ready [upgrade to PHP 7.1](https://github.com/nextras/migrations/pull/130) baseline. I hope to finish that, merge this and release on weekend.

Reply:
OK, closing this issue.

Reply:
It seems that it is about PHP 7.1, not Symfony 7 support.